### ISSUE TITLE
Binary folder

### ISSUE THREAD

**Issue by A-thaVan** [EVerest/everest-core#472]
Is it recommended that we copy binaries from `build/dist/bin` to `~/.local/bin` since it has already been added to the path?

If so, should it be reflected in the documentation?

**Comment by corneliusclaussen** [comment]
No, this is not really recommended. If you want to install somewhere else you should add -DCMAKE_INSTALL_PREFIX=/my/target/dir to cmake and then install using cmake install. This will ensure the library paths are correct
